### ISSUE TITLE
Major comments

### ISSUE THREAD

**Issue by Alanocallaghan** [VallejosGroup/BASiCSWorkflow#43]
# Code-related
- [x] Update R version
- [x] Use droplet data instead, with more heterogeneity?
- [x] logit transform volcano plot?
- [ ] Demonstrate reasons behind prior choice/prior sensitivity checks (PPC)   

> Alan: I think prior predictive checks of the regression model don't work because of the inverse gamma on scale, so I'll probably just put some text in about the prior justification and refer to the old papers.   
> Cata: I agree that PPC are likely to fail. Supp Text 3 of the original BASiCS paper shows some sensitivity wrt to the hyper-parameters used for $\theta$ and $\delta_i$. Of course, those results are rather outdated as the prior for $\delta_i$ now follows the approach in Eling et al. 

- [ ] Multiple chains? Rhat diagnostic.  

> Alan: **I am working on this at the moment between a new package and BASiCS itself**
  BASiCstan is now on Bioc. Will implement single chain Rhat for BASiCS and say maybe we'll do multiple chains in future but not now.
> Cata: Thanks for uploading BASiCSstan to BioC!. For this paper, I think we can argue that the sampler is rather time-consuming and therefore it's unlikely that users will routinely run multiple chains. We can also mention that we have generally observed the sampler behaves well in a large number of contexts. However, the ability to run multiple chains will improve once we publish the scalability paper (which we should try to submit soon!)

- [ ] How to manipulate/visualise credible intervals?

> Alan: BASiCS_Summary I guess
> Cata: Yes, that would give you credible intervals for individual parameters. Should we edit the BASiCS_Test function to also return HPD intervals for LFCs? That should be easy to implement. 

- [ ] Simplify and reduce code chunks (particularly plotting); plotting is verbose (less customisation maybe)   

> Alan: This is tough, might just ditch some figures and make some a bit more "ugly"
  eg for the sake of a workflow having ugly axis labels is probably okay? need to check other f1000 papers

# Text-related
- [x] What type of experiments/experimental designs?

https://github.com/VallejosGroup/BASiCSWorkflow/blob/a40c412e5d20e6f3bb2e8214450a67ce761a85d7/Workflow.Rmd#L147-L155

- [ ] Better model summary (with schematics)
  - [ ] Explain joint prior
  - [ ] explain variance decomp better
  
- [x] How does it deal with biological replicates?

> Cata: it does not. That's scive. :-)
 
- [ ] Computationally challenging - so why not just MLE or MAP?

> Cata: We can argue against a simple MLE based on an e.g. NB model because our main focus is on estimates of variability. Inference about means is simpler and, indeed, an MLE approach is likely sufficient (at some point I remember comparing point estimates with DESeq2, etc). This is not the case for over-dispersion parameters. Here is where the use of Bayesian approach becomes more useful. In particular, we have seen that the joint mean/dispersion prior (aka the regression approach) improve inference of transcriptional variability for lowly expressed genes and sparse datasets. In terms of MAP, I think the argument to give is that we want to retain posterior uncertainty to enable differential testing. We can add a couple of sentences about this when describing the method. 

- [ ] Volcano plot probs seem to bunch around 0.5, why? Are they calibrated?
- [x] How does it work with spatial data?

> Cata: We have not tested that. 

- [ ] Address double dipping

> Cata: Whilst we highlight that this is an important issue, we have not addressed it yet. 

- [x] What happens with violation of model assumptions?
- [ ] Dealing with a BASiCS run per population (muscat approach...? not sure what that refers to)
- [x] Divide and conquer MCMC/other computational approaches for speed - downsampling, filtering

# Manuscript-related
- [ ] Better navigation (eg linking to sections more)
- [ ] make available as bioc/github workflow with better navigation  
  **Response**: I would say "We'll do this with a later/final version". Actually think this is the best way to maintain it to ensure it still runs in later versions of Bioc etc...

**Comment by catavallejos** [comment]
I am closing this as most things have been addressed already in the rebuttal. I will start a new issue to track what remains unresolved.
